trunk/SUAVE/Analyses/Stability/AVL.py

```diff
@@ -323,48 +326,51 @@ def sample_training(self):
         Cm_alpha = np.zeros([len(AoA)*len(mach),1])
         Cn_beta  = np.zeros([len(AoA)*len(mach),1]) 
         NP       = np.zeros([len(AoA)*len(mach),1]) 
+
+        # Calculate aerodynamics for table
```

Comment:
This looks a lot like the aerodynamics methods. Do you think it's duplicative to have both?